src/video_core/vulkan_common/vulkan_device.cpp

```diff
@@ -587,6 +587,13 @@ Device::Device(VkInstance instance_, vk::PhysicalDevice physical_, VkSurfaceKHR
             ext_extended_dynamic_state = false;
         }
     }
+    if (ext_sampler_filter_minmax) {
+        if (driver_id == VK_DRIVER_ID_AMD_PROPRIETARY ||
+            driver_id == VK_DRIVER_ID_AMD_OPEN_SOURCE || driver_id == VK_DRIVER_ID_MESA_RADV) {
```

Comment:
Following the style of the rest of the function
```suggestion
    const bool is_amd = driver_id == VK_DRIVER_ID_AMD_PROPRIETARY ||
                        driver_id == VK_DRIVER_ID_MESA_RADV ||
                        driver_id == VK_DRIVER_ID_AMD_OPEN_SOURCE;
    if (ext_sampler_filter_minmax && is_amd) {
```